pkg/config/default.go

```diff
@@ -242,11 +242,7 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 	c.ImageDefaultTransport = _defaultTransport
 	c.StateType = BoltDBStateStore
 
-	c.OCIRuntime = "runc"
-	// If we're running on cgroupv2 v2, default to using crun.
-	if cgroup2, _ := cgroupv2.Enabled(); cgroup2 {
-		c.OCIRuntime = "crun"
-	}
+	c.OCIRuntime = "crun"
```

Comment:
as much as I like to see runc go away, should we have the reverse of the old if statement turning runc on if cgroups isn't enabled?  Or at least the test and a message saying crun is not currently enabled?

Reply:
These are just defaults, so if user or distro puts something in containers.conf it will override.  There is no situation where we want the default to be runc, so setting the default to always be crun is fine.

Reply:
I'm fine with not defaulting, but should we put up warnf here if crun is not found to be enabled?  I.e. keep the old if, but instead of dropping into runc instead throw a warning?

Reply:
I think us not finding `crun` in this case is a hard error, at least in Podman land. We should consider either making this default conditional on `crun` being installed, or making Podman switch defaults if the default runtime is not available - I'd prefer the former.